martin/src/cog/mod.rs

```diff
@@ -306,18 +324,9 @@ fn get_meta(path: &PathBuf) -> Result<Meta, FileError> {
         zoom_and_tile_across_down.insert(zoom, (tiles_across, tiles_down));
     }
 
-    let min_zoom = zoom_and_ifd
-        .keys()
-        .min()
-        .ok_or_else(|| CogError::NoImagesFound(path.clone()))?;
-
-    let max_zoom = zoom_and_ifd
-        .keys()
-        .max()
-        .ok_or_else(|| CogError::NoImagesFound(path.clone()))?;
     Ok(Meta {
-        min_zoom: *min_zoom,
-        max_zoom: *max_zoom,
+        min_zoom: 0,
+        max_zoom: images_ifd.len() as u8 - 1,
```

Comment:
Is `NoImagesFound` not an error that can happen?

If `images_ifd.len()==0` this seems like buggy logic.

Reply:
Oops I made a mistake here. It could happen but rarely. Thx!!  :D